snapshots/helpers_test.go

```diff
@@ -7,17 +7,17 @@ import (
 	"crypto/sha256"
 	"errors"
 	"io"
-	"os"
 	"testing"
 	"time"
 
 	protoio "github.com/gogo/protobuf/io"
 	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/libs/log"
 	db "github.com/tendermint/tm-db"
 
 	"github.com/cosmos/cosmos-sdk/snapshots"
-	"github.com/cosmos/cosmos-sdk/snapshots/types"
-	snapshottypes "github.com/cosmos/cosmos-sdk/snapshots/types"
+	snapshotypes "github.com/cosmos/cosmos-sdk/snapshots/types"
```

Comment:
Please revert this to "snapshottypes" which isn't a naming stutter and won't unnecessarily create diffs too.

Reply:
Made a typo when batch renaming, thanks for catching that